gnome-initial-setup/demo/eos-demo-video.c

```diff
@@ -106,8 +121,13 @@ create_demo_window (GisDemoVideo *demo_video)
                            demo_video,
                            G_CONNECT_AFTER);
 
+  g_signal_connect_object (priv->main_window,
```

Comment:
Will this not make it so that if someone e.g. alt-tabs back and forth the pipeline state will be set again? Not sure if it's a problem in practice.

Reply:
Yeah, I think that'd be a good idea.

Reply:
In my testing, this change of signal does not help.

FWIW, as soon as I hit the Alt button, the video window is closed.

Reply:
Yeah, the video does still not start at the beginning with this change. Starts at a similar time then before which suggests that the delay is introduced by another factor.